tests/final/etc-writable-symlinks.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+echo "Test that the symlinks for etc/{timezone,localtime,hostname} exist"
+set -x
+for f in timezone localtime hostname; do
```

Comment:
i think at least timezone should be a symlink to the UTC timezone that should be available in the core20 snap.

Reply:
Thanks, I updated the 030-fix-timedatectl.chroot which already sets /etc/timezone (but not localtime). This should be ready for a re-review now.